active/0000-remove-ember-cli-eslint.md

```diff
@@ -0,0 +1,51 @@
+- Start Date: 2018-08-13
+- RFC PR: (leave this empty)
+
+# Summary
+
+Remove https://github.com/ember-cli/broccoli-lint-eslint +
+https://github.com/ember-cli/ember-cli-eslint as a dependency within
+`ember-cli`.
```

Comment:
Are you proposing the removal of lint hooks completely from ember-cli, or just removing `ember-cli-eslint` as a default dependency of new projects?

Reply:
As far as I understand, only reason why we need the lint hooks is to trigger logic within `ember-cli-eslint`. (I assumed they were tightly coupled since they were marked as private).

If that's not the case, we should leave the link hooks as is.

Reply:
There are some hooks beyond the ones you mentioned here, like [`lintTree`](https://ember-cli.com/api/classes/Addon.html#method_lintTree) that are used by [a number of addons in the ecosystem](https://emberobserver.com/code-search?codeQuery=lintTree%28).

But more broadly, `broccoli-lint-eslint` and `ember-cli-eslint` aren't dependencies of `ember-cli` itself, only of projects that the CLI generates. Given the phrasing here, I'm still a little unclear about whether you're proposing that Ember CLI stop supporting linters as part of the build, or just changing the default for new projects while leaving the ability for folks to opt in 🙂

Reply:
I can see how that's unclear. I'll make some changes 👍 Thanks for the feedback @dfreeman